JsonSubTypes.Tests/DemoKnownSubTypeWithProperty.cs

```diff
@@ -40,6 +40,19 @@ public void Demo()
             Assert.AreEqual("Painter", (persons.Last() as Artist)?.Skill);
         }
 
+        [Test]
+        public void DemoDifferentCase()
+        {
+            string json = "[{\"Department\":\"Department1\",\"JobTitle\":\"JobTitle1\",\"FirstName\":\"FirstName1\",\"LastName\":\"LastName1\"}," +
+                          "{\"Department\":\"Department1\",\"JobTitle\":\"JobTitle1\",\"FirstName\":\"FirstName1\",\"LastName\":\"LastName1\"}," +
+                          "{\"skill\"" +
+                          ":\"Painter\",\"FirstName\":\"FirstName1\",\"LastName\":\"LastName1\"}]";
+
+
+            var persons = JsonConvert.DeserializeObject<ICollection<Person>>(json);
+            Assert.AreEqual("Painter", (persons.Last() as Artist)?.Skill);
```

Comment:
I'm suggesting to add yet another Artist with `Skill` to show that both cases are valid

Reply:
It's already there on [line 34](https://github.com/manuc66/JsonSubTypes/pull/39/files/493d48564587cd9878a02154fbe4d543d790261f#diff-bb2cbc8ae9988a0a27793302e2dd0d64R34)

Reply:
Ah ok, I have only seen the diff.